include/gluon_ffi_interface/sc_gluon_v1_entry_points.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include "sc_gluon_v1_types.h"
+
+#ifdef __cplusplus
+#    define SC_GLUON_EXTERN extern "C"
+#endif
+
+#define SC_GLUON_EXPORT SC_GLUON_EXTERN __attribute__((__visibility__("hidden")))
```

Comment:
Shouldn't this be public/default?


```suggestion
#define SC_GLUON_EXPORT SC_GLUON_EXTERN __attribute__((__visibility__("default")))
```

Reply:
yes, thanks!